packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -1,52 +1,114 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import withStyles from '../styles/withStyles';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import { emphasize } from '../styles/colorManipulator';
 import MoreHorizIcon from '../internal/svg-icons/MoreHoriz';
 import ButtonBase from '../ButtonBase';
+import breadcrumbCollapsedClasses, {
+  getBreadcrumbCollapsedUtilityClass,
+} from './breadcrumbCollapsedClasses';
 
-const styles = (theme) => ({
-  button: {
-    display: 'flex',
-    marginLeft: theme.spacing(0.5),
-    marginRight: theme.spacing(0.5),
-    backgroundColor: theme.palette.grey[100],
-    color: theme.palette.grey[700],
-    borderRadius: 2,
-    '&:hover, &:focus': {
-      backgroundColor: theme.palette.grey[200],
-    },
-    '&:active': {
-      boxShadow: theme.shadows[0],
-      backgroundColor: emphasize(theme.palette.grey[200], 0.12),
-    },
+const overridesResolver = (props, styles) => {
+  return deepmerge(styles.root || {}, {
+    [`& .${breadcrumbCollapsedClasses.button}`]: styles.button,
+    [`& .${breadcrumbCollapsedClasses.icon}`]: styles.icon,
+  });
+};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes } = styleProps;
+
+  const slots = {
+    root: ['root'],
+    button: ['button'],
+    icon: ['icon'],
+  };
+
+  return composeClasses(slots, getBreadcrumbCollapsedUtilityClass, classes);
+};
+
+const BreadcrumbCollapsedRoot = experimentalStyled(
+  'li',
+  {},
+  {
+    name: 'PrivateBreadcrumbCollapsed',
+    slot: 'Root',
+    overridesResolver,
+  },
+)({});
+
+const BreadcrumbCollapsedButton = experimentalStyled(
+  ButtonBase,
+  {},
+  {
+    name: 'PrivateBreadcrumbCollapsed',
+    slot: 'Button',
   },
-  icon: {
-    width: 24,
-    height: 16,
+)(({ theme }) => ({
+  display: 'flex',
+  marginLeft: theme.spacing(0.5),
+  marginRight: theme.spacing(0.5),
+  backgroundColor: theme.palette.grey[100],
+  color: theme.palette.grey[700],
```

Comment:
A side note

<img width="204" alt="Capture d’écran 2021-01-21 à 18 36 05" src="https://user-images.githubusercontent.com/3165635/105388967-904e8400-5c17-11eb-8f22-af4a949c68b8.png">

It's not dark mode friendly